docs/source/contributing.rst

```diff
@@ -28,9 +226,13 @@ it in the toctree in ``docs/source/index.rst``. (Toctree = table of contents
 tree.) The toctree will preserve the order given.
 
 You can check your work by asking whoever maintains these docs (as of 2022,
-Kevin Fu) to add your branch as a version on the ReadTheDocs dashboard. They
-should go to the webpage, add a version that points to the new doc development
-branch, and build that version, then give you the link.
+Kevin Fu) to add your branch as a version on the ReadTheDocs dashboard.
+
+.. note::
+
+    If you are maintaining these docs, you must install the requirements in ``docs/requirements.txt``.
```

Comment:
you actually don't need to unless you're building locally with `make docs`, can you add a new section that explains local building?